### ISSUE TITLE
Error loading entries

### ISSUE THREAD

**Issue by Moji14** [Neverous/efibooteditor#83]
I get this errors once I run efibooteditor:

![Screenshot](https://github.com/Neverous/efibooteditor/assets/30596071/44cbe9a0-ec4b-4ddb-89be-9a1961d7f2fd)

**Comment by Neverous** [comment]
It seems like you have references to non-existing entries in the `BootOrder`, maybe you can check what `efibootmgr` sees? If those entries indeed don't exist you should be fine ignoring this error and saving will just clean up the `BootOrder` EFI variable.
Can you share RAW dump (`Help -> Dump raw EFI data`)? It should show the same error but dump the other data it was able to access nevertheless.

**Comment by Moji14** [comment]
efibootmgr see Boot2002 or Boot2004

mou@R2D2:~$ efibootmgr
BootCurrent: 0002
Timeout: 5 seconds
BootOrder: 0000,3000,0002,0003,2001,2002,2004
Boot0000* Windows Boot Manager
Boot0002* ubuntu
Boot0003* kali
Boot2001* EFI USB Device
Boot3000* Internal Hard Disk or Solid State Disk

Dump raw EFI data throws the next:

{
    "Boot": {
        "0000": {
            "efi_attributes": 7,
            "raw_data": "AQAAAHQAVwBpAG4AZABvAHcAcwAgAEIAbwBvAHQAIABNAGEAbgBhAGcAZQByAAAABAEqAAEAAAAACAAAAAAAAAAgAwAAAAAAwpjIkyPnZUGkNx+MxzV6kAICBARGAFwARQBGAEkAXABNAGkAYwByAG8AcwBvAGYAdABcAEIAbwBvAHQAXABiAG8AbwB0AG0AZwBmAHcALgBlAGYAaQAAAH//BABXSU5ET1dTAAEAAACIAAAAeAAAAEIAQwBEAE8AQgBKAEUAQwBUAD0AewA5AGQAZQBhADgANgAyAGMALQA1AGMAZABkAC0ANABlADcAMAAtAGEAYwBjADEALQBmADMAMgBiADMANAA0AGQANAA3ADkANQB9AAAAbwABAAAAEAAAAAQAAAB//wQA"
        },
        "0002": {
            "efi_attributes": 7,
            "raw_data": "AQAAAGIAdQBiAHUAbgB0AHUAAAAEASoAAQAAAAAIAAAAAAAAACADAAAAAADCmMiTI+dlQaQ3H4zHNXqQAgIEBDQAXABFAEYASQBcAHUAYgB1AG4AdAB1AFwAcwBoAGkAbQB4ADYANAAuAGUAZgBpAAAAf/8EAA=="
        },
        "0003": {
            "efi_attributes": 7,
            "raw_data": "AQAAAF4AawBhAGwAaQAAAAQBKgABAAAAAAgAAAAAAAAAIAMAAAAAAMKYyJMj52VBpDcfjMc1epACAgQEMABcAEUARgBJAFwAawBhAGwAaQBcAGcAcgB1AGIAeAA2ADQALgBlAGYAaQAAAH//BAA="
        },
        "2001": {
            "efi_attributes": 7,
            "raw_data": "AQAAAAQARQBGAEkAIABVAFMAQgAgAEQAZQB2AGkAYwBlAAAAf/8EAFJD"
        },
        "3000": {
            "efi_attributes": 7,
            "raw_data": "AQAAAAQASQBuAHQAZQByAG4AYQBsACAASABhAHIAZAAgAEQAaQBzAGsAIABvAHIAIABTAG8AbABpAGQAIABTAHQAYQB0AGUAIABEAGkAcwBrAAAAf/8EAFJD"
        }
    },
    "BootCurrent": {
        "efi_attributes": 6,
        "raw_data": "AgA="
    },
    "BootOrder": {
        "efi_attributes": 7,
        "raw_data": "AAAAMAIAAwABIAIgBCA="
    },
    "OsIndications": {
        "efi_attributes": 7,
        "raw_data": "AAAAAAAAAAA="
    },
    "OsIndicationsSupported": {
        "efi_attributes": 6,
        "raw_data": "HwAAAAAAAAA="
    },
    "SecureBoot": {
        "efi_attributes": 6,
        "raw_data": "AA=="
    },
    "SetupMode": {
        "efi_attributes": 6,
        "raw_data": "AA=="
    },
    "Timeout": {
        "efi_attributes": 7,
        "raw_data": "BQA="
    },
    "VendorKeys": {
        "efi_attributes": 6,
        "raw_data": "AQ=="
    },
    "_Type": "raw"
}

**Comment by Neverous** [comment]
Yeah, efibootmgr also doesn't see them so its probably some leftovers that BIOS didn't care to clean up?
Unless in BIOS boot menu you see some records after `EFI USB device` / see more than 5 options. Otherwise you should be good to ignore that error and it should disappear after saving (as it would overwrite the `BootOrder` with only available entries).
